tasks/main.yml

```diff
@@ -1,19 +1,50 @@
 ---
-  - name: Update apt-cache
-    apt: update_cache=yes 
+- name: Update apt-cache
+  apt:
+    update_cache: yes
 
-  - name: Install curl
-    apt: 
-      name:   ['curl']
-      state: latest
+- name: Install curl
+  apt:
+    name: curl
+    state: present
 
-  - name: set apt sources
-    shell: curl -sL https://deb.nodesource.com/setup_19.x | sudo bash -
+- name: Set apt sources for Node.js
+  shell: curl -sL https://deb.nodesource.com/setup_19.x | bash
```

Comment:
I suppose if we are going to update this repo we should update it to Node 20.x ?